v3.1/reference/alert-rules.md

```diff
@@ -435,6 +435,10 @@ Emergency-level alerts are often caused by a service or node failure. Manual int
 
     `sum(increase(tidb_tikvclient_gc_action_result{type="success"}[6h])) < 1`
 
+    > **Note:**
+    >
+    > In TiDB 3.* versions, the `tidb_tikvclient_gc_action_result` metric exists but does not have a value. It's because garbage collection (GC) is introduced in TiDB 3.* versions but will not be performed in TiDB.
```

Comment:
```suggestion
    > In TiDB 3.* versions, the `tidb_tikvclient_gc_action_result` metric exists but does not have a value. It's because distributed garbage collection (GC) is introduced in the TiDB 3.0 version but will not be performed in TiDB.
```